test/source/tests/decrypt.ts

```diff
@@ -1366,15 +1347,13 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
     test(
       `verify - sha1 shows error`,
       testWithBrowser('compatibility', async (t, browser) => {
-        t.mockApi!.configProvider = new ConfigurationProvider({
-          attester: {
-            pubkeyLookup: {
-              'sha1@sign.com': {
-                pubkey: sha1signpubkey,
-              },
+        t.mockApi!.configProvider!.config!.attester = {
```

Comment:
@rrrooommmaaa, do you prefer setting options like this?
I prefer using the `ConfigurationProvider` constructor and passing the config there.

This is because in the near future, we will also have to pass ekm, wkd, and other configs to configProvider, and if 
we don't pass it to constructor we would end up force-unwrapping `configProvider!.config!.objectToConfig` for each config, which is not ideal.